x/claim/keeper/claim_test.go

```diff
@@ -162,3 +162,40 @@ func TestSetClaimRecord(t *testing.T) {
 	err = keeper.SetClaimRecord(ctx, claimRecord)
 	require.NoError(t, err)
 }
+
+func TestGetAllClaimRecords(t *testing.T) {
+	keeper, ctx := testkeeper.ClaimKeeper(t)
+
+	addr1 := sdk.AccAddress(secp256k1.GenPrivKey().PubKey().Address()).String()
```

Comment:
we could write a function like `GetRandomArkeoAddr()` , and then reuse it everywhere